lib/transaction/trace/index.js

```diff
@@ -224,7 +224,7 @@ Trace.prototype.getTotalTimeDurationInMillis = function getTotalTimeDurationInMi
   while (segments.length !== 0) {
     const segment = segments.pop()
     totalTimeInMillis += segment.getExclusiveDurationInMillis()
-    segment.getChildren().forEach((childSegment) => segments.push(childSegment))
+    segments.push(...segment.getChildren())
```

Comment:
It looks like we have a test that creates a lot of child segments and then contains stacks.  This test is now failing by spreading the children and pushing vs iterating and pushing